routes/users.js

```diff
@@ -1662,18 +1648,16 @@ router.get('/user/:id/renew', async function(req, res){
                 return plugin_call(plugin_info, user.uid, user, session_user.uid);
                 // eslint-disable-next-line no-unused-vars
             })).then(function(results){
-                return send_notif(mailOptions, fid, []);
+                res.send({message: 'Activation in progress', fid: fid, error: []});
+                res.end();
+                return;
             }, function(err){
-                return send_notif(mailOptions, fid, err);
```

Comment:
same as previous, mail sent before plugin execution and possible failure

Reply:
now send mail and plugins are unlinked so they should not failed one because of the other